How to install debug adapter

I'm trying to install but it fails:
```
λ swipl -g "pack_install(debug_adapter)" -t halt                                                                                                          
% Contacting server at https://www.swi-prolog.org/pack/query ... ok                                       
Install debug_adapter@0.2.5 from https://github.com/eshelyaron/debug_adapter/releases/download/v0.2.5/debug_adapter-0.2.5.zip Y/n?                             
                                                                                                        
% Contacting server at https://www.swi-prolog.org/pack/query ... ok                                                                   
% "debug_adapter-0.2.5.zip" was downloaded 6 times                                                      
Package:                debug_adapter                                                                                                 
Title:                  Debug Adapter Protocol package for SWI-Prolog                                                                 
Installed version:      0.2.5                                                                                                         
Author:                 Eshel Yaron <eshelshay.yaron@gmail.com>                                                                       
Maintainer:             Eshel Yaron <eshelshay.yaron@gmail.com>                                                                       
Packager:               Eshel Yaron <eshelshay.yaron@gmail.com>                                                                         
Home page:              https://github.com/eshelyaron/debug_adapter                                                                   
Download URL:           https://github.com/eshelyaron/debug_adapter/releases/download/v0.2.5/debug_adapter-0.2.5.zip                    
Install "debug_adapter-0.2.5.zip" (91,667 bytes) Y/n?                                                                                                        
% checking for swipl...                                                                                                               
% /usr/lib/swipl/bin/x86_64-linux/swipl                                                                                               
% checking for a thread-safe mkdir -p...                                                                                                                     
% /usr/bin/mkdir -p                                                           
% checking for a BSD-compatible install...                                                                                                                   
% /usr/bin/install -c                                                                                                                                        
% checking whether build environment is sane...                                                                                                                
% yes                                                                          
% checking for gawk...                                                         
% gawk                                                                         
% checking whether make sets $(MAKE)...                                        
% yes                                                                          
% checking whether make supports nested variables...                                                                                                           
% yes                                                                          
% checking that generated files are newer than configure...                                                                                                    
% done                                                                         
% configure: creating ./config.status                                          
% config.status: creating pack.pl                                              
% config.status: creating Makefile                                             
% config.status: creating prolog/Makefile                                      
% config.status: creating test/Makefile                                        
% config.status: creating test/server.plt                                      
%  cd . && /bin/sh /home/risto/.local/share/swi-prolog/pack/debug_adapter/missing automake-1.16 --foreign                                                      
ERROR: configure.ac:5: error: version mismatch.  This is Automake 1.16.5,                                                                                      
ERROR: configure.ac:5: but the definition used by this AM_INIT_AUTOMAKE                                                                                        
ERROR: configure.ac:5: comes from Automake 1.16.1.  You should recreate                                                                                        
ERROR: configure.ac:5: aclocal.m4 with aclocal and run automake again.                                                                                         
Warning: prolog/Makefile.am:2: warning: addprefix $(srcdir: non-POSIX variable name                                                                            
Warning: prolog/Makefile.am:2: (probably a GNU make extension)                                                                                                 
Warning: prolog/Makefile.am:4: warning: user target 'swipl_debug_adapter$(EXEEXT)' defined here ...                                                            
Warning: /usr/share/automake-1.16/am/program.am: ... overrides Automake target 'swipl_debug_adapter$(EXEEXT)' defined here                                     
Warning: prolog/Makefile.am:1:   while processing program 'swipl_debug_adapter'                                                                                
% WARNING: 'automake-1.16' is probably too old.                                                                                                                
%          You should only need it if you modified 'Makefile.am' or                                                                                            
%          'configure.ac' or m4 files included by 'configure.ac'.                                                                                              
%          The 'automake' program is part of the GNU Automake package:                                                                                         
%          <https://www.gnu.org/software/automake>                                                                                                             
%          It also requires GNU Autoconf, GNU m4 and Perl in order to run:                                                                                     
%          <https://www.gnu.org/software/autoconf>                                                                                                             
%          <https://www.gnu.org/software/m4/>                                                                                                                  
%          <https://www.perl.org/>                                             
% make: *** [Makefile:283: Makefile.in] Error 1                                                                                                                
ERROR: -g pack_install(debug_adapter): Process "process(path(make),[all])": exit status: 2                                                    
```

Is there a way to get it working without autotools?

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -25,7 +25,6 @@ jobs:
           ./configure
           make
           make check
-          make distcheck
 
       - name: Build pack
         run: |

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -27,4 +27,3 @@ jobs:
           ./configure
           make
           make check
-          make distcheck

--- a/Makefile.am
+++ b/Makefile.am
@@ -1,2 +1,8 @@
 SUBDIRS = prolog test
 EXTRA_DIST = pack.pl
+
+dist-hook:
+	rm $(top_distdir)/configure.ac
+	rm $(top_distdir)/Makefile.am
+	rm $(top_distdir)/prolog/Makefile.am
+	rm $(top_distdir)/test/Makefile.am

